client.js

```diff
@@ -6,17 +6,18 @@ const options = {
   ui: 'bdd',
 };
 
-if (Meteor.settings.public["MOCHA_TIMEOUT"]) {
-  options.timeout = Meteor.settings.public["MOCHA_TIMEOUT"];
-}
-
-let config = {}
 
 // Attempt to load config from .mocharc file
 try {
-  config = JSON.parse(__meteor_runtime_config__['meteortesting:mocha-core_config'])
+  const config = JSON.parse(__meteor_runtime_config__['meteortesting:mocha-core_config'])
+  options = { ...options, ...config };
```

Comment:
I guess this will throw an exception, because you cannot reassign a value to a constant - the exception is ignored and nobody is informed about that something happened ...

Please introduce at least a warning if an exception is thrown which is written out on the console - can also be written to `stderr` if feasible.

I also thought that this could be streamlined with the implementation on the server.